riff-raff/app/views/schedule/form.scala.html

```diff
@@ -0,0 +1,44 @@
+@(configForm: Form[controllers.ScheduleController.ScheduleForm], prismLookup: resources.PrismLookup)(implicit request: Security.AuthenticatedRequest[AnyContent, com.gu.googleauth.UserIdentity], messages: Messages)
+@import b3.vertical.fieldConstructor
+@import helper.CSRF
+
+@main("Schedule", request, List("form-autocomplete")) {
+
+    <h2>Schedule Configuration</h2>
+    <hr/>
+
+    @b3.form(action=routes.ScheduleController.save) {
+        @CSRF.formField
+        @snippets.inputHidden(configForm("id"))
+
+        @if(configForm.hasGlobalErrors) {
+            <div class="alert alert-danger">
+                <h4>Error</h4>
+                <ul>
+                @configForm.globalErrors.map { error =>
+                    <li>@error.message</li>
+                }
+                </ul>
+            </div>
+        }
+
+        @b3.text(configForm("projectName"), 'id -> "projectInput", Symbol("data-url") -> "/deployment/request/autoComplete/project", '_label -> "Project Name")
+        @b3.select(
+            configForm("stage"),
+            options = helper.options(prismLookup.stages.toList),
+            '_default -> "--- Choose a stage ---",
+            '_label -> "Stage",
+            '_error -> configForm.globalError.map(_.withMessage("Please select deployment stage"))
+        )
+        @b3.text(configForm("schedule"), '_label -> "Schedule expression")
+        <p>This should be a weird Quartz cron expression
```

Comment:
Be good to give a (couple of) example(s) here. Most teams will be doing similar things and the docs are pretty dense.

Reply:
Done. Think one is enough.

Reply:
There are a few traps we talked about here, which could happen by accident or deliberately:

* people scheduling it too frequently
* scheduling overnight/weekends
* everyone scheduling at the same time

If the feature proves useful, we'll probably need to address some or all of these.